### DISCUSSION TITLE
Configure done data of invoked services (machines)

### DISCUSSION THREAD

**Discussion by geekox86** [statelyai/xstate#3311] [Q&A] [answered]
Greetings,

I like how actions, services, etc. can be configured while keeping the machine definition a serializable object with no logic (functions).

Is it possible to do the same with done data of invoked services (machines) rather than hard coding their functions?

For example, using a string name of the data function.

**Comment by ChrisShank** [comment, marked as answer]
That cant be externalized at the moment.

**Comment by Andarist** [comment]
Ye, at the moment it's not possible but we plan to re-evaluate all of the things that can't be externalized right now. I'd like to have a clear story for all of them.
